draft-ietf-quic-qpack.md

```diff
@@ -1025,7 +1025,7 @@ entry.
 ### Indexed Field Line
 
 An indexed field line representation identifies an entry in the static table,
-or an entry in the dynamic table with an absolute index less than the value of
+or an entry in the dynamic table with an relative index less than the value of
```

Comment:
No; the Base is an absolute index.  An Indexed Field Line of this type uses relative indices, and so can only reference a certain region of the table; this is describing which region.